xdsl/irdl.py

```diff
@@ -1265,18 +1265,17 @@ def from_pyrdl(pyrdl_def: type[ParametrizedAttribute]) -> ParamAttrDef:
 
         return ParamAttrDef(name, parameters)
 
+    def verify(self, attr: ParametrizedAttribute):
+        """Given an IRDL definition, verify that an attribute satisfies its invariants."""
```

Comment:
This comment is out of date now, right?

Reply:
I also think the method name here could use a change. Maybe something like "adheres to"? To say that the the attr satisfies the irdl definition.

Reply:
Updated, ty. I think we can stick with the current name for the time being, we use verify everywhere to mean the same thing.